css-layout-classes/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title> CSS Layout Classes</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+
+  <body>
+    <div class="container">
+      <section class="row">
+        <header> <h1>Choose your character! </h1> </header>
+      </section>
+      <section class="row">
+        <div class="picture-column">
+          <img src="images/link.png" alt="Link">
+        </div>
+        <div class="text-column">
+          <p>
+          <h2>Super Smash Bros. Ultimate</h2>
+          Super Smash Bros. Ultimate Recieved universal acclaim from both critics and player with
+          some critis calling it the best installment in the series. It recieved praise for its
+          large amount of content and fine tuning of existing Smash gameplay elements, although
+          its online mode was widely critized.
+          
+        </div>
+        <div class="picture-column">
+          <img src="images/yoshi.png" alt="Yoshi">
+        </div>
+      </section>
+      <section class="row">
+        <div class="column-1">
+          <img src="images/kirby.png" alt="Kirby">
+        </div>
+        <div class="column-2">
+          <img src="images/fox.png" alt="StarFox">
+        </div>
+        <div class="column-3">
+          <img src="images/pikachu.png" alt="Pikachu">
+        </div>
+        <div class="column-4">
+          <img src="images/ness.png" alt="Ness">
+        </div>
```

Comment:
Since these columns are all 1/4th the width of the row, try making just one class that they can all use.